tedana/workflows/tedana.py

```diff
@@ -738,7 +738,12 @@ def tedana_workflow(
         else:
             comptable = pd.read_table(ctab)
 
-            if manacc is not None:
+            # If manacc is None and if comptable has a column named original_classification
+            if manacc is None and "original_classification" in comptable.columns:
+                # Get index of rows that have accepted on their classification column in comptable
+                manacc = comptable.index[comptable["classification"] == "accepted"].tolist()
+                comptable, metric_metadata = selection.manual_selection(comptable, acc=manacc)
```

Comment:
I don't think we can assume that an updated component table reflects manual classification. For example, if we get AROMA working, we could run the same data through both tedana and AROMA to produce merged classifications derived from both workflows.

Reply:
Okay, would you be happy if we updated the rationale column instead?

Reply:
Hopefully once we have the decision tree refactor done and have switched from rationales to the messages we plan to use, we can just require classification tools to update that information. In the mean time, I think you empty string solution is great. On RICA's side, could you update the rationale column as well?

Reply:
I agree things will be different soon, but users need a fix now.

Re Rica: I could make Rica update the rationale to say "manual" or something similar.

Reply:
Yes, if Rica could updated the rationale for any component with an updated classification, that would be optimal. "Manual" or "I001" would both work well.

Reply:
[PR #42 in Rica](https://github.com/ME-ICA/rica/pull/42) adds this functionality.